### PR TITLE
Networks Update

### PR DESCRIPTION
**Pull request by rachelhox**
## Description

Should I add `heightSocket` info so that I could add the new networks to `animated network` on Stake Now and Home pages?
e.g.
https://github.com/forbole/official-website/blob/e3259f32b0bac6ba0181db0ff34d0356820b88f5/src/utils/network_info.ts#L19
@kwunyeung 

## Checklist
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: src/utils/network_info.ts [hunk 1]
```diff
@@ -261,6 +261,36 @@ const logos = {
...
+    image: "/static/images/icons/injective-protocol.png",
+    name: "Injective Portocol",
+    key: "injective-protocol",
+  },
+  straightedge: {
+    image: "/static/images/icons/straightedge.png",
+    name: "Straightedge",
+    key: "straightedge",
+  },
+  tachyon: {
```
Comment (kwunyeung) [inline src/utils/network_info.ts:274]: @rachelhox please remove Tachyon, Near and OpenLibra.

**Review by kwunyeung — CHANGES_REQUESTED** [review]

**Comment by kwunyeung** [comment]
Looks like you need to update https://github.com/forbole/official-website/blob/master/src/screens/home/components/supported_networks/config.ts in order to show them on the home page.

**Comment by rachelhox** [comment]
> @rachelhox Looks like you need to update https://github.com/forbole/official-website/blob/master/src/screens/home/components/supported_networks/config.ts in order to show them on the home page.

so I'd just show them the icons? not the height socket info?

**Comment by kwunyeung** [comment]
Just show the icons now. Seems on-going maintenance is very troublesome if we query the heights for every network. We may not have endpoints for some of the networks.

**Comment by rachelhox** [comment]
> Just show the icons now. Seems on-going maintenance is very troublesome if we query the heights for every network. We may not have endpoints for some of the networks.

I'll just hide the height socket info for this component then!

**Review by kwunyeung — APPROVED** [review]
ACK

**Pull request merged**